MensaMenuWidget/MensaService.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  MensaService.swift
+//  TUM Campus App
+//
+//  Created by Nikolai Madlener on 12.12.21.
+//  Copyright © 2021 TUM. All rights reserved.
+//
+
+import Foundation
+
+final class MensaService {
+    static let shared = MensaService()
+
+    private init() {}
+    
+    public func getMensaMenu(mensaApiKey: String) async throws -> Menu? {
+        let url = URL(string: "https://tum-dev.github.io/eat-api/" + "\(mensaApiKey)/\(Date().year)/\(Date().weekOfYear).json")!
```

Comment:
Our API model can handle this already:
```suggestion
          let url = try EatAPI.menu(location: location, year: Date().year, week: Date().weekOfYear).asURLRequest().url
```